### PR TITLE
[IBA-305] Add Tenant.beta_tester

### PR DESCRIPTION
**Pull request by eprudnikov**
It adds a very simple API to fetch and update a tenant's beta tester identifier. The idea is to have a simple marker telling us who can access beta features (e.g. the first integrations).
BTW, if you have a better name for that identifier, please suggest it. I found other options like beta_program_participant too wordy.

### CODE REVIEW

**Review by aehrenthal — APPROVED** [review]

**Review by olyavertwist — APPROVED** [review]
Looks good. Alternatively, we can use "early users" instead of beta testers.

File: hub/hub-api.yaml [hunk 1]
```diff
@@ -2328,6 +2328,9 @@ components:
           description: The list of features available to the tenant.
           items:
             type: string
+        betaTester:
```
Comment (BullDog57Rus) [inline hub/hub-api.yaml:2331]: let's add this field as required? and also we can add it to all other fields where needed :D

**Review by BullDog57Rus — APPROVED** [review]
